index.html

```diff
@@ -66,6 +66,199 @@ <h3>I'm a Software Developer</h3>
         </ul>
       </div>
     </main>
+    <section class="projects">
+     <div class="recent-project">
+        <h2 class="my-recent-work">My Recent Works</h2>
+        <hr class="my-work">
+        <img class="recent-project-img" src="./icons/Recentwork image.svg" alt="project image">
+        <div class="project-description">
+        <h2 class="recent-project-name">Multi-Post Stories</h2>
+        <div class="recent-project-note">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="recent-project-stack">
+            <ul class="recent-project-tech">
+              <li class="recent-project-tech-used">CSS</li>
+              <li class="recent-project-tech-used">HTML</li>
+              <li class="recent-project-tech-used">Bootstrap</li>
+              <li class="recent-project-tech-used">Ruby</li>
+            </ul>
+          </div>
+          <button class="see-recent-project">See Project</button>
+        </div>
+      </div> 
+      </div>
+      
+      <div class="projects-card">
+       <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
```

Comment:
- [ ] Kindly make sure the button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=SU8lUrhODQH8Lc5d-0)) as this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) of the project 

> Implement the button interactions (enable, hover, etc.).

<kbd>
<img width="296" alt="Screenshot 2023-04-30 at 2 05 08 AM" src="https://user-images.githubusercontent.com/44798044/235324255-52f77572-9e89-4363-b38f-30ebc88a6ba6.png">
</kbd>